config/software/openssl.rb

```diff
@@ -0,0 +1,165 @@
+name "openssl"
```

Comment:
We should be able to update `omnibus-software` to 5b6eb618d1cce161d62b49b2a684164aafbee828 in `Gemfile.lock`, and then use something like #257 to override it in the Ruby configs, right? There should be no need to copy the entire file…

Reply:
Resolved offline — the approach I suggested triggers a cascade of other errors, as we are bumping to a much newer version of `omnibus-software`.

Reply:
Sync'd offline. To summarize this:

We initially attempted to upgrade omnibus to a version that includes the hash of openssl `1.1.1q` (at that point At which point we can simply override it in `config/projects/google-fluentd.rb`), but ran into various failures and down a rabbit hole because we are too far behind on omnibus version (The one that has openssl 1.1.1q is `22.8.XX`. We are still at `4.0.0`).

In order to pick up the CVE fixes earlier, we decided to go with the current implementation.